JSIL.Libraries/Includes/Bootstrap/Core/Classes/System.Convert.js

```diff
@@ -83,6 +83,15 @@ JSIL.ImplementExternals("System.Convert", function ($) {
     return result;
   };
 
+  function toBase64CharArray(inArray, offsetIn, length, outArray, offsetOut, options) {
+      var tempArray = JSIL.StringToCharArray(toBase64StringImpl(inArray, offsetIn, length, options));
```

Comment:
Can you change toBase64StringImpl to take an output array instead so you don't need a temporary string like this?